### PR TITLE
fix missing lines in d3 #8

### PR DESCRIPTION
**Pull request by jogal360**

### CODE REVIEW

File: index_d3.html [hunk 1]
```diff
@@ -10,29 +10,32 @@
 		<script src="interactions_d3.js"></script>
 -->
 		<script type="text/javascript">//<![CDATA[ 
-			function fetchJSONFile(path, callback) {
+			unction fetchJSONFile(path, callback)
```
Comment (remyla) [inline index_d3.html]: mistype

**Comment by remyla** [comment]
I will need to review the code with you tomorrow - I will need this addNode method later on - thanks, good evening

**Comment by jogal360** [comment]
Problems with brackets, corrected. Here in operation:
http://jogal360.github.io/graphViewer/

Do you want to add the addNode method?

**Comment by remyla** [comment]
To be exact, the addNode method was not to be removed. it is important for the next steps of development

**Comment by jogal360** [comment]
New version check the files changed.
Here in operation:
http://jogal360.github.io/graphViewer/

**Comment by jogal360** [comment]
Fixed problems. 
My pull request is open
Here in operation:
http://jogal360.github.io/graphViewer/

**Comment by remyla** [comment]
integrated in graph_d3.js your code on lines 169 and 170 to fix the bug (separate the calls to d3 methods data() and the enter()). Thanks @jogal360 for this bugfix

**Pull request closed without merging**